autofill.gypi

```diff
@@ -50,6 +50,7 @@
           '<(libchromiumcontent_dir)/signin_core_browser.lib',
           '<(libchromiumcontent_dir)/signin_core_common.lib',
           '<(libchromiumcontent_dir)/os_crypt.lib',
+          'crypt32.lib',
```

Comment:
`-` to support specifying such items as `-llib`, meaning “library lib in the library search path.” referenced from https://gyp.gsrc.io/docs/InputFormatReference.md